tests/test_directives/test_directive_other.py

```diff
@@ -136,6 +136,18 @@ def test_reversed_toctree(app):
                 includefiles=['baz', 'bar/index', 'foo'])
 
 
+@pytest.mark.sphinx(testroot='toctree-glob')
+def test_toctree_class(app):
+    text = (".. toctree::\n"
+            "   :class: custom-toc\n"
+            "\n"
+            "   foo\n")
```

Comment:
```suggestion
    text = ('.. toctree::\n'
            '   :class: custom-toc\n'
            '\n'
            '   foo\n')
```

Reply:
Done. But note that all neighboring examples use double quotes.